src/System.Collections.Immutable/tests/ImmutableSortedDictionaryTest.cs

```diff
@@ -425,8 +425,8 @@ public static void TestDebuggerAttributes_Null()
             Assert.IsType<ArgumentNullException>(tie.InnerException);
         }
 
-        ////[Fact] // not really a functional test -- but very useful to enable when collecting perf traces.
-        public void EnumerationPerformance()
+        // [Fact] // not really a functional test -- but very useful to enable when collecting perf traces.
+        private void EnumerationPerformance()
```

Comment:
💡 To distinguish this case from other helpers, it would be preferable to use `[Fact(Skip = "explanation")]` rather than mark it private.